packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -446,12 +464,14 @@ module.exports = function (log, config) {
     };
   };
 
-  Mailer.prototype.send = function (message) {
+  Mailer.prototype.send = async function (message) {
     log.trace(`mailer.${message.template}`, {
       email: message.email,
       uid: message.uid,
     });
-    const localized = this.localize(message);
+    isNewEmails = message.template === 'welcome' ? true : false;
```

Comment:
I know the issue intends to localize the `cadReminder` template but since I have the other two PRs pending to be merged, I thought it'd be worth testing the overall working of this functionality on a test template.